src/components/sections/Catalogue/CurrentProductSection/components/Articles/ProductDescription/components/ReviewsBlock.tsx

```
@@ -0,0 +1,52 @@
+'use client'
+
+import useEmblaCarousel from 'embla-carousel-react'
+
+export default function ReviewsBlock({
+  reviews,
+  isReviewsLoading,
+  isReviewsError
+}: {
+  reviews?: any[]
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)